### PR TITLE
TASK: Fix spelling mistakes in ``FeatureList`` documentation

### PR DESCRIPTION
**Pull request by michaelgerdemann**
This fixes some spellings in the FeatureList

### CODE REVIEW

File: TYPO3.Neos/Documentation/GettingStarted/FeatureList.rst [hunk 1]
```diff
@@ -86,6 +86,6 @@ Surf Deployment and Cloud Support
 =================================
 
 The developers of Neos also created "Surf" a professional tool for downtime free server-deployment that is optimzed for
-Neos. With Surf Neos can be easily deployed to all kinds of hosting environments beein it dedicated servers, virtual-machines
+Neos. With Surf Neos can be easily deployed to all kinds of hosting environments be in it dedicated servers, virtual-machines
```
Comment (aertmann) [inline TYPO3.Neos/Documentation/GettingStarted/FeatureList.rst]: I think the original intention was "being"

Comment (michaelgerdemann) [inline TYPO3.Neos/Documentation/GettingStarted/FeatureList.rst]: Oh yes, you are right... 🙈

**Comment by aertmann** [comment]
:+1:

**Comment by aertmann** [comment]
StyleCI issue is irrelevant and fixed (branch needs to be rebased to pass)

:+1:

**Pull request merged**